config/env/default.js

```diff
@@ -24,8 +24,8 @@ module.exports = {
   },
   // sessionSecret should be changed for security measures and concerns
   sessionSecret: process.env.SESSION_SECRET || 'MEAN',
-  // sessionKey is set to the generic sessionId key used by PHP applications
-  // for obsecurity reasons
+  // sessionKey is the session name and is set to the generic sessionId key
+  // used by PHP applications for obsecurity reasons
```

Comment:
I understand the point - it helps to give as little information about your framework as possible in order to minimize automated attacks. In this case the objective appears to be to save cookies that look like cookies saved by PHP frameworks. However, after a quick Google I couldn't find a documented practice of using "sessionId" as a default name for PHP cookies. Looking through my own cookies, every website seemed to use a different name for session id anyway. 

I don't think this should be a security concern and perhaps the comment should be removed to prevent confusion.